### PR TITLE
Fix for the failure of trigger e2e test

### PR DESCRIPTION
**Pull request by daisy-ycguo**
## Proposed Changes

* Remove labels and triggers before ending the e2e test

### CODE REVIEW

File: test/e2e/trigger_test.go [hunk 1]
```diff
@@ -40,13 +40,17 @@ func TestBrokerTrigger(t *testing.T) {
 
 	err = test.lableNamespaceForDefaultBroker(t)
 	assert.NilError(t, err)
+	defer unlableNamespaceForDefaultBroker(t)
```
Comment (navidshaikh) [inline test/e2e/trigger_test.go]:
```suggestion
	defer test.unlableNamespaceForDefaultBroker(t)
```

**Comment by daisy-ycguo** [comment]
@navidshaikh will you check if you are OK with the changes to trigger e2e test ? I removed label and deleted triggers before shutting down the test environment to avoid flaky errors.

File: test/e2e/trigger_test.go [hunk 2]
```diff
@@ -58,11 +62,19 @@ func TestBrokerTrigger(t *testing.T) {
 	test.verifyTriggerDescribe(t, r, "updtrigger", "default", "sinksvc0", []string{"a", "b"})
 	test.triggerUpdate(t, r, "updtrigger", "type=knative.dev.bar", "sinksvc1")
 	test.verifyTriggerDescribe(t, r, "updtrigger", "default", "sinksvc1", []string{"a", "b", "type", "knative.dev.bar"})
+	test.triggerDelete(t, r, "updtrigger")
 
 	t.Log("create trigger with error return")
 	test.triggerCreateMissingSink(t, r, "errtrigger", "notfound")
 }
 
+func (test *e2eTest) unlableNamespaceForDefaultBroker(t *testing.T) {
```
Comment (matzew) [inline test/e2e/trigger_test.go]: /lgtm
/approve

**Comment by navidshaikh** [comment]
@daisy-ycguo : Yes, lets merge this and unblock other PRs.

FYI @dsimansk logged https://github.com/knative/eventing/issues/2668 for this issue.

**Pull request merged**